Examples/Algorithms/Geant4/src/SensitiveSteppingAction.cpp

```diff
@@ -132,11 +132,14 @@ void ActsExamples::SensitiveSteppingAction::UserSteppingAction(
   // Get the physical volume & check if it has the sensitive string name
   const G4VPhysicalVolume* volume = track->GetVolume();
   if (volume == nullptr) {
-    ACTS_ERROR("No volume found for track " << track->GetTrackID());
-    std::terminate();
```

Comment:
I think @paulgessinger just added this - but I guess nothing speaks against an exception?

Reply:
@paulgessinger Is there a specific reason for a `std::terminate` here?

Reply:
I don't remember a specific reason, so let's use an exception.